packages/account/src/Sections/Security/ClosingAccount/closing-account-general-error-content.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Button } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TClosingAccountGeneralErrorContentProps = {
+    message: string;
+    onClick: () => void;
+};
+
+const ClosingAccountGeneralErrorContent = ({ message, onClick }: TClosingAccountGeneralErrorContentProps) => (
+    <React.Fragment>
+        <div className='closing-account-error__container closing-account-error__container-message'>
+            <div className='closing-account-error__details closing-account-error__details-message'>{message}</div>
+        </div>
+        <div>
+            <Button className='closing-account-error__button' primary onClick={onClick}>
```

Comment:
Do we need this div?

```
<React.Fragment>
        <div className='closing-account-error__container closing-account-error__container-message'>
            <div className='closing-account-error__details closing-account-error__details-message'>{message}</div>
        </div>
        
           <Button className='closing-account-error__button' primary onClick={onClick}>
                <Localize i18n_default_text='OK' />
            </Button>
    </React.Fragment>
```

Reply:
for the Button's `float:right` styling to work, a wrapper is required. removing the div is causing the button to be displayed left.